lib/phlex.rb

```diff
@@ -9,6 +9,14 @@ module Phlex
 	#  rescue Phlex::Error
 	module Error; end
 
+	if defined?(ERB::Escape)
+		Escape = ERB::Escape
+	elsif defined?(ERB::Util)
+		Escape = ERB::Util
+	else
+		Escape = BasicEscape
```

Comment:
I ended up removing this because you need to require ERB anyway, and all versions of ERB have `ERB::Util`.